docs/api.md

```diff
@@ -651,34 +611,56 @@ const logger2 = pino(pino.extreme())
 `dest` can be either a file or a file descriptor. If it is omitted, it
 will be `process.stdout.fd`.
 
-`pino.extreme()` is implemented on [`sonic-boom`](https://github.com/mcollina/sonic-boom).
+`pino.extreme()` is implemented with the [`sonic-boom` ⇗](https://github.com/mcollina/sonic-boom)
+module.
 
-<a id="metadata"></a>
-# Metadata
+* See [Extreme mode ⇗](extreme.md).
+* See [`sonic-boom` ⇗](https://github.com/mcollina/sonic-boom)
```

Comment:
You should also point to pino-multi-stream

Reply:
ok

Reply:
hold on I think you've read the removed lines - this isn't the metadata section (that was moved under destination option), this is the pino-extreme secion. I've added a link to pino-multi-stream from there instead

Reply:
this has been address https://github.com/pinojs/pino/pull/447/files#diff-0f426376b964e609c5c4618eb8bed3b2R227 and https://github.com/pinojs/pino/pull/447/files#diff-0f426376b964e609c5c4618eb8bed3b2R245